torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace torch {
+namespace jit {
+
+struct UpgraderEntry {
+  int version_bump;
+  std::string upgrader_name;
+  std::string old_schema;
+};
+
+static std::unordered_map<std::string, std::vector<UpgraderEntry>> operator_version_map(
+    {{"aten::div.Tensor",
+      {{4,
+        "div_Tensor_0_3",
+        "aten::div.Tensor(Tensor self, Tensor other) -> Tensor"}}},
```

Comment:
Why old schema is needed here?

Reply:
I need it to compile the older models with their specific schema and replace it with the upgraders later.